msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -208,6 +208,21 @@ The app manifest describes how the app integrates into the Microsoft Teams platf
         "id": "AAD App ID",
         "resource": "Resource URL for acquiring auth token for SSO"
     },
+    "graphConnector": {
+        "notificationUrl": "The URL where Graph-connector notifications for the application must be sent"
```

Comment:
"notificationUrl": "The URL where Graph connector notifications for the application must be sent"